spec/models/miq_worker/runner_spec.rb

```diff
@@ -38,4 +38,35 @@
       expect(@worker_base.worker_monitor_drb).to eq 0
     end
   end
+
+  context "#get_messages" do
+    before do
+      allow_any_instance_of(MiqWorker::Runner).to receive(:worker_initialization)
+      @worker_base = MiqWorker::Runner.new
```

Comment:
Possibly you could have added this to the top of the spec:

```ruby
module MiqWorkerForSpecs
  module Runner < MiqWorker::Runner
    def initialize(cfg ={})
    end
  end
end
```

Then these two lines could have changed to this single line:

```ruby
@worker_base = MiqWorkerForSpecs::Runner.new
```